agent/stats/queue_test.go

```diff
@@ -30,6 +30,9 @@ const (
 	// predictableHighMemoryUtilizationInMiB is the expected Memory usage in MiB for
 	// the "predictableHighMemoryUtilizationInBytes" value (7377772544 / (1024 * 1024))
 	predictableHighMemoryUtilizationInMiB = 7035
+	// the "predictableInt64Overflow" requires 3 metrics to guarantee overflow
+	// note math.MaxInt64 is odd, so integer division will trim off 1
+	predictableInt64Overflow = math.MaxInt64 / int64(2)
```

Comment:
nit: this can just be `math.MaxInt64 / 2`

Reply:
when I do this I get:
`constant 4611686018427387903 overflows int`
the cast is necessary here